### DISCUSSION TITLE
How to remove an element from plydata['vertex']

### DISCUSSION THREAD

**Discussion by JACMoutinho** [dranjan/python-plyfile#64] [Q&A]
I was wondering if you could me with something, not really an issue:

I want to delete certain values from a point cloud using plyfile, like the example i have bellow where i want to delete the coordinate with index 567890 so that i can later produce a new ply file without that point.

![image](https://github.com/dranjan/python-plyfile/assets/80013048/317f9e58-9d52-4c20-a74a-195b60fa4d89)

How would ago about that?

Thank you for your help and have a wonderful weekend.

**Comment by dranjan** [comment]
The way I would approach this is to operate on the underlying NumPy array using NumPy methods (perhaps [numpy.delete](https://numpy.org/doc/stable/reference/generated/numpy.delete.html)) and then reassign the result to the `.data` attribute, something like:

```python
plydata.elements[0].data = numpy.delete(plydata.elements[0].data, [567890])
```

I expect that isn't particularly efficient, but in any case the general strategy illustrated here is what I would suggest: get the NumPy array using `some_element.data`, make the changes you want, and then reassign the result with `some_element.data = ...`.
